portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,29 @@
 // limitations under the License.
 
 async function getPhrase(){
-    let response = await fetch('/load-data');
-    let list = await response.json();
-    if(list.length != 0){
-        for(i = 0; i < list.length; i++){
-            let commentSection = document.createElement("DIV");
-            commentSection.setAttribute("id", "dynamic-history");
-            let addName = document.createTextNode(list[i].name + ": ");
-            let addMessage = document.createTextNode(list[i].message);
-            commentSection.appendChild(addName);  
-            commentSection.appendChild(addMessage);  
-            document.getElementById("section").appendChild(commentSection);
-        }
+    let max = document.getElementById("max-comments").value;
+    let response = await fetch('/load-data?numCommentsDisplay=' + max);
+    let list = await response.json(); //list of entities from datastore
+    let commentHistory = document.getElementById("section"); //UI for displaying comments
+
+    while (commentHistory.hasChildNodes()) {
+        commentHistory.removeChild(commentHistory.childNodes[0]);
     }
+
+    for(i = 0; i < list.length; i++){
```

Comment:
If you are only using the indexes to get each list element, can you do `for...of` style loop?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of